app/src/main/java/com/batch/mcs/finalproject/CalendarDisplayFragment.java

```diff
@@ -0,0 +1,65 @@
+package com.batch.mcs.finalproject;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link CalendarDisplayFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class CalendarDisplayFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
```

Comment:
Also, this is default code.... let's try to avoid injecting unnecessary code... I will tell you tomorrow why